README.md

```diff
@@ -10,11 +10,20 @@ See the example env file for more info.
 Once you've properly setup the .env file, you can run the `start_compose.sh` script:
 ```
 Usage:
-./start_compose.sh [--dev | --prod] (any other args to pass to docker, e.g. --build)
+./start_compose.sh [--dev | --prod | --build | --setup-aws]
```

Comment:
At line 200 in app.py it is set to "--setupaws". not sure whether it matters, but it may be better to have it in same format.